### ISSUE TITLE
Why cant it be libraries like this that get popular?

### ISSUE THREAD

**Issue by TrevorBlythe** [yixuan/MiniDNN#23]
:( 
Its better than the others.

**Comment by Asheeshkrsharma** [comment]
@TrevorBlythe 
Fair point. Perhaps you would like to join us in closing some of the issues? :)

**Comment by rkuang9** [comment]
> :( Its better than the others.

What do you think about this one https://github.com/rkuang9/FLARE?
